jaeger-core/src/main/java/io/jaegertracing/internal/Constants.java

```diff
@@ -12,7 +12,7 @@
  * the License.
  */
 
-package io.jaegertracing;
+package io.jaegertracing.internal;
```

Comment:
some of these technically could be considered public, e.g. 
  * TRACER_IP_TAG_KEY - there have been requests to allow providing IP manually via tracer tag.
  * SAMPLER_TYPE_TAG_KEY, SAMPLER_PARAM_TAG_KEY - analytics jobs often need to read and interpret these span tags
  * DEBUG_ID_HEADER_KEY - feels like public, but not sure what the exact use case is.

Reply:
I'll open a new follow-up PR with a new constants class, to discuss and move the candidates.

Reply:
https://github.com/jaegertracing/jaeger-client-java/issues/473